src/shared/utils/getMessageAttributes.ts

```diff
@@ -0,0 +1,18 @@
+import { _throw } from "../../shared/utils/_throw";
+
+export function getMessageAttributes(msg: Element, className: string): { [key: string]: string } {
+    const element = msg.querySelector(`.${className}`);
```

Comment:
not a fan of doing this inside this function, this function should only extract the attributes. 
Is there a reason this is done in here and not *before* calling this function?

Reply:
Absolutly not. It was just to have fewer lines in the other 3 files to have less duplicate code. Ill move it again to the main page